packages/opentelemetry-host-metrics/test/metric.test.ts

```diff
@@ -314,15 +317,15 @@ describe('Host Metrics', () => {
     it('should export Process CPU time metrics', async () => {
       const metric = await getRecords(reader, 'process.cpu.time');
 
-      ensureValue(metric, { state: 'user' }, 90713.56);
-      ensureValue(metric, { state: 'system' }, 63192.630000000005);
+      ensureValue(metric, { state: 'user' }, 90.71356);
```

Comment:
Note for reviewer: values have just changed in the magnitude since according to [NodeJS docs](https://nodejs.org/api/process.html#processcpuusagepreviousvalue) the usage is reported in microseconds and not milliseconds